apps/condo/k6/src/syncTourStep.test.ts

```diff
@@ -0,0 +1,72 @@
+import { check } from 'k6'
+
+import {
+    setupCondoAuth,
+    createOrganization,
+    createProperty,
+    sendAuthorizedRequest,
+    getOrganizationEmployeeId, createTicket, createMeter,
+} from './utils'
+
+const DURATION = '60s'
+
+export const options = {
+    tags: { testid: 'tour', serverUrl: __ENV.BASE_URL },
+    scenarios: {
+        syncTourSteps: {
+            exec: 'syncTourSteps',
+            executor: 'constant-arrival-rate',
+            duration: DURATION,
+            rate: 5,
+            timeUnit: '1s',
+            preAllocatedVUs: 7,
+        },
+    },
+    thresholds: {
+        http_req_failed: ['rate<0.01'],
+        http_req_duration: ['med<2000'],
+    },
+}
+
+export function setup () {
+    const { token } = setupCondoAuth(true)
+
+    const createdOrganization = createOrganization({ token })
+    const organizationId = createdOrganization.json('data.obj.id')
+
+    getOrganizationEmployeeId({ token, organizationId })
+    const createdProperty = createProperty({ token, organizationId })
+
+    const data = {
+        token,
+        organizationId,
+        propertyId: createdProperty.json('data.obj.id'),
+    }
+
+    for (let i = 0; i < 200; i++) {
+        createTicket(data)
+        createMeter(data)
+    }
```

Comment:
Should you add a billing entities test here? This seems was the request that took the longest to process.